spec/models/manageiq/providers/vmware/infra_manager/inventory/collector_spec.rb

```diff
@@ -66,7 +66,7 @@
         assert_ems
 
         host = RbVmomi::VIM.HostSystem(vim, "host-41")
-        allow(host).to receive(:collect!).and_return(nil)
```

Comment:
This was causing the `host_guest_devices.count` spec to fail because we weren't collecting the `host.config.storageDevice` inventory during targeted refresh.

The solution was just to return the same set of storage devices as the full refresh